File: help/pt_pt/exercicios_da_ficha_3/Ficha03_018/Ficha03_018.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
#define length( a ) ( sizeof( a ) / sizeof( a[0] ))
//Main Function
int main() {
    printf("Número positivo\t: ");
    int num;
    scanf("%d", &num);
    printf("Número de dígitos \t= %d",length( num + "" ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:32

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_018/Ficha03_018.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_018 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Número positivo\t: " );
        int num = Integer.valueOf(keyb.nextLine());
        System.out.print("Número de dígitos \t= " + ( num + "" ).length());
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:32
